plugins/logitech-hidpp/fu-logitech-hidpp-device.c

```diff
@@ -177,8 +176,8 @@ fu_logitech_hidpp_device_ping(FuLogitechHidppDevice *self, GError **error)
 
 	/* format version in BCD format */
 	if (priv->hidpp_version != FU_HIDPP_VERSION_BLE) {
-		version = (gdouble)msg->data[0] + ((gdouble)msg->data[1]) / 100.f;
-		priv->hidpp_version = (guint)version;
+		/* minor version is in msg->data[1] */;
```

Comment:
Surely you still want to keep it though don't you?

Reply:
I don't think we care about the minor -- it's really to check if we're running version 1 or 2. @hardboprobot do you remember why we care about the minor?

Reply:
Got it, agree -- thanks!